### ISSUE TITLE
Mobile

### ISSUE THREAD

**Issue by rubenmarcus** [haskellcamargo/gemidao-do-zap#14]
Tem planos de fazer um app pra ios/android? Se sim ,tem algo que eu posso ajudar?? (código, visual etc)

**Comment by haskellcamargo** [comment]
Sim, tava pensando nisso! Talvez um que tu possa também selecionar contatos da agenda para simular e enviar. A API é bem simples de chamar, então seria tranquilo. O que seria mais trabalhoso seria a parte visual. Aliás, podemos (vamos!) planejar. :D

**Comment by rubenmarcus** [comment]
Po tenho interesse em ajudar, posso bolar um UX/UI com as telas e até um logo se quiser, mas queria meter a mão no código também. tipo sei lá vi q vc ta fazendo em React, tentar usar React Native sei lá.

**Comment by williamokano** [comment]
Já criaram o repo?

**Comment by yannbf** [comment]
Opa, tô dentro! React native? Partiu

**Comment by rubenmarcus** [comment]
um problema da logistica é que o ideal seria fazer um bot q pega o numero do cara e faz o cadastro ja da token, pq o usuario leigo nao vai querer fazer esse passo, só pra dar aquela zuada

**Comment by williamokano** [comment]
Tutorial em vídeo, sucesso!

**Comment by rubenmarcus** [comment]
ok, vamos falar nas telas então desse app.

Serão quantas? 3? 4?

1 - Tela de login/ apresentação?
2 - Tela de Envio
3 - Tela de Ajuda ??
4 - Tela de Cadastro de Token ??
5 - Tela de Histórico de envios ??

**Comment by williamokano** [comment]
Loin acho que não precisa

Em 8 de ago de 2017 5:16 PM, "Ruben Marcus" <notifications@github.com>
escreveu:

**Comment by rubenmarcus** [comment]
Então acho q dá pra colocar a tela do totalvoice no proprio app...

**Comment by rubenmarcus** [comment]
acho que a galera ia dar downvote por ser só 10 free kkkk mas acho q o app ia bombar

**Comment by williamokano** [comment]
Ahuahaus, mas se eles quiserem mais eles que paguem, 10 já dá pra zoar
maneiro, ainda mais lendo agenda hauahahaha

Em 8 de ago de 2017 5:19 PM, "Ruben Marcus" <notifications@github.com>
escreveu:

**Comment by williamokano** [comment]
Anyway, alguém com conhecimentos de jogos? Fazer tipo uma roleta russa. A gente força o cara se cadastrar com "número real" e faz um jogo "Escape the gemidão" que vai ser mais um runner dentro os milhões que existem, com a diferença que cada vez que ele morrer o número dele vai enviar um gemidão pra um número aleatório da agenda hUASehuaSeuhAHUse

Só pra gerar a discórdia mesmo!

**Comment by StanleySathler** [comment]
"Escape the gemidão" foi o melhor UAHUAHUAHUA.

Sinceramente, acho que pra ficar bem simples, uma tela é o suficiente. Um campo para o token, dois para os contatos. E uma de ajuda, para guiar a galera nos passos iniciais. Pra um MVP, na minha opinião, já tá legal.

**Comment by walteraa** [comment]
@haskellcamargo eu comentei com um amigo meu ontem, sobre um serviço mobile. Rola subir um form web também, se quiser posso fazer o form web e subir em um container em um cluster da gcloud. :v:

**Comment by williamokano** [comment]
Web já temos 2 versões, mas é sempre bem vindo

Em 9 de ago de 2017 2:01 PM, "Walter Alves" <notifications@github.com>
escreveu:

**Comment by rubenmarcus** [comment]
@williamokano  cade a url do web?

**Comment by williamokano** [comment]
@rubenmarcus não tenho agora, vi no grupo da APDA. Mas tem um gemidão as a service que fizeram tbm, `gemid.in`, mas não está opensouce... ainda. @greenboxal vamos liberar esse fonte aí

**Comment by SkyaTura** [comment]
Faz um runner que se o cara errar ele manda um gemidão pra algum contato aleatorio com o numero dele pela API do Total Voice kkkkkkk

**Comment by williamokano** [comment]
Foi o que eu disse

Em 9 de ago de 2017 8:57 PM, "Skya Tura" <notifications@github.com>
escreveu:

**Comment by SkyaTura** [comment]
Nossa mano, me ignora, isso é falta de café kkkkkkk

Foi mal

**Comment by williamokano** [comment]
Kkkkk, de boa mano hauahaahauahua

**Comment by guicolares** [comment]
Olá Galera, posso fazer para iOS com Swift, só em Android que to sem tempo. Só me entregar o layout e as chamados do Rest

**Comment by darlanxw** [comment]
fiz uma versão tosca em swift pra iOS, só pra distribuir pra alguns amigos que não são dev's kkk, se alguém quiser, segue: https://github.com/darlanxw/gemidao-do-zap-ios-app

**Comment by rogermfranco** [comment]
Como pego essa versão pra ios ja compilada... tbm não sou dev? Mas to participando da zueira kkkkk

**Comment by darlanxw** [comment]
Passa seu email que eu libero pra ti via testflight

**Comment by rogermfranco** [comment]
@darlaxw segue meu email rogermfranco@gmail.com (gemidão vai dominar o mundo) huahauhauaha
